src/index.js

```diff
@@ -158,7 +161,7 @@ f.pinMode = function (pin, direction, mux, pullup, slew, callback) {
                 } else
                     callback(err, false);
             }
-            return (false);
+            // return (false);		// Continue for now
```

Comment:
Didn't you find a fix in the code I added above this?  I don't think this is necessary.

Reply:
Yup, it works now.